install_config/aggregate_logging.adoc

```diff
@@ -1335,23 +1334,24 @@ config.yaml: |
   .defaults:
     delete:
       days: 31
-    runhour: 0
-    runminute: 0
-    timezone: America/New_York
 
   .regex:
     - pattern: '^project\..+\-dev\..*$'
       delete:
-        days: 15
+        days: 1
     - pattern: '^project\..+\-test\..*$'
       delete:
-        days: 30
+        days: 2
 ----
 
+[NOTE]
```

Comment:
I might move this note to line 1252 and s/timezone/Curator time zone (The CCS glossary says "time zone" is preferred.)